lib/services/elasticsearch.js

```diff
@@ -687,6 +687,62 @@ class ElasticSearch extends Service {
     }
   }
 
+  /**
+  * Updates all documents matching the provided filters
+  *
+  * @param {String} index - Index name
+  * @param {String} collection - Collection name
+  * @param {Object} query - Query to match documents
+  * @param {Object} changes - Changes wanted on documents
+  * @param {Object} options - from (undefined), size (undefined), refresh (undefined), timeout (undefined), userId (undefined)
+  *
+  * @returns {Promise.<Object>} { documents, total, failures: [ _shardId, reason ] }
+  */
+  async updateByQuery(
+    index,
+    collection,
+    query,
+    changes,
+    { refresh, from, size } = {})
+  {
+
+    try {
+      const esRequest = {
+        index: this._getESIndex(index, collection),
+        body: this._sanitizeSearchBody({ query }),
+        scroll: '5s',
+        from,
+        size
+      };
+
+      const documents = await this._getAllDocumentsFromQuery(esRequest);
+      documents.forEach(document => {
+        document._source = null;
+        document.body = changes;
+        return document;
+      });
+      
+      const response = await this.mUpdate(
+        index,
+        collection,
+        documents,
+        { refresh }
+      );
+
+      debug('Update by query: %o', esRequest);
+
+      await this.refreshCollection(index, collection);
+
+      return {
+        documents: response.items,
+        total: response.items.length,
```

Comment:
This property is useless since we have the `documents` array, so we can use the `length` property